app/styles/global.css

```diff
@@ -15,23 +15,27 @@ body {
   margin: 0;
   font-family: "Poppins", sans-serif;
   line-height: 1.6;
-  font-size: 18px;
 }
 
-* {
+/* * {
   box-sizing: border-box;
-}
+} */
+
+/* img {
+  max-width: 100%;
+  display: block;
+} */
 
-a {
-  color: #0070f3;
-  text-decoration: none;
+h1 {
```

Comment:
please dont target html tags, always use classes

Reply:
I deleted this code in the next pull requests, this code was from the nextjs tutorial.

Reply:
I want to apply these style for h1 in all pages, that's why I targeted in this way.